src/braket/aws/aws_device.py

```diff
@@ -103,6 +108,7 @@ def run(
         poll_timeout_seconds: float = AwsQuantumTask.DEFAULT_RESULTS_POLL_TIMEOUT,
         poll_interval_seconds: Optional[float] = None,
         inputs: Optional[Dict[str, float]] = None,
+        gate_calibrations: Optional[GateCalibrations] = None,
```

Comment:
I think it might make sense for this to be just the (Gate, Qubit) mapping to the PulseSequence. I'm not sure what the value is in forcing the users to nest this inside another object.

Reply:
Ack, will update